test/ci/cpu_unit_test_linux.yml

```diff
@@ -11,7 +11,7 @@ trigger: none
 
 jobs:
 - job: cpu_unit_tests_linux
-  timeoutInMinutes: 40 # how long to run the job before automatically cancelling
+  timeoutInMinutes: 120 # how long to run the job before automatically cancelling
```

Comment:
sorry it needs to be increased in the other file, the build name/file name were reversed, I just fixed them now